File: BackEnd/ajax/Usuario.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Usuario.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Usuario.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'login',
    'correo': null,
    'contrasena': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Usuario.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'registrar',
    'idPais': null,
    'noIdentidad': null,
    'nombreCentroMedico': null,
    'pNombre': null,
    'sNombre': null,
    'sApellido': null,
    'idTipoUsuario': null,
    'direccion': null,
    'correo': null,
    'idTipoCentroMedico': null,
    'pApellido': null,
    'contrasena': null,
    'sexo': null,
    'direccionCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/Usuario.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Usuario'
data={
  'accion':'listarTipos',
}
print postear(data,url)
print

url='Usuario'
data={
  'accion':'login',
  'correo': None,
  'contrasena': None,
}
print postear(data,url)
print

url='Usuario'
data={
  'accion':'registrar',
  'idPais': None,
  'noIdentidad': None,
  'nombreCentroMedico': None,
  'pNombre': None,
  'sNombre': None,
  'sApellido': None,
  'idTipoUsuario': None,
  'direccion': None,
  'correo': None,
  'idTipoCentroMedico': None,
  'pApellido': None,
  'contrasena': None,
  'sexo': None,
  'direccionCentroMedico': None,
}
print postear(data,url)
print
